3 Failures to build,  Errors.

**Describe the bug**

3 failures.
ci/circleci: build-debian-arm64-11-bullseye - Your tests failed on CircleCI
[Details](https://circleci.com/gh/rgleason/weather_routing_pi/9816)
ci/circleci: build-debian-armhf-11-bullseye - Your tests failed on CircleCI
[Details](https://circleci.com/gh/rgleason/weather_routing_pi/9804)
ci/circleci: build-debian-x86_64-11-bullseye - Your tests failed on CircleCI
[Details](https://circleci.com/gh/rgleason/weather_routing_pi/9809)


--------------------
build-debian-armhf-11-bullseye
https://app.circleci.com/pipelines/github/rgleason/weather_routing_pi/711/workflows/b03e53b1-5196-4a4d-87e6-c6db4352370f/jobs/9804
```
iles/weather_routing_pi.dir/src/RouteMapOverlay.cpp.o -c /ci-source/src/RouteMapOverlay.cpp
/ci-source/src/RouteMap.cpp:139:10: error: 'std::function' has not been declared
  139 |     std::function<double(double)> postProcessFn = nullptr) {
      |          ^~~~~~~~
/ci-source/src/RouteMap.cpp:139:18: error: expected ',' or '...' before '<' token
  139 |     std::function<double(double)> postProcessFn = nullptr) {
      |                  ^
/ci-source/src/RouteMap.cpp:139:5: error: default argument missing for parameter 7 of 'double GetWeatherParameter(RouteMapConfiguration&, double, double, const wxString&, int, double, int)'
  139 |     std::function<double(double)> postProcessFn = nullptr) {
      |     ^~~~~~~~~~~~~
```

-------
Build Debian build-debian-x86_64-11-bullseye (9809)  fails.  https://app.circleci.com/pipelines/github/rgleason/weather_routing_pi/711/workflows/b03e53b1-5196-4a4d-87e6-c6db4352370f/jobs/9809

```
-fno-strict-aliasing  -O2 -s  -DPREFIX=\"/usr/local\" -std=gnu++11 -o CMakeFiles/weather_routing_pi.dir/src/RouteMapOverlay.cpp.o -c /home/circleci/project/src/RouteMapOverlay.cpp
/home/circleci/project/src/RouteMap.cpp:139:10: error: ‘std::function’ has not been declared
  139 |     std::function<double(double)> postProcessFn = nullptr) {
      |          ^~~~~~~~
/home/circleci/project/src/RouteMap.cpp:139:18: error: expected ‘,’ or ‘...’ before ‘<’ token
  139 |     std::function<double(double)> postProcessFn = nullptr) {
      |                  ^
/home/circleci/project/src/RouteMap.cpp:139:5: error: default argument missing for parameter 7 of ‘double GetWeatherParameter(RouteMapConfiguration&, double, double, const wxString&, int, double, int)’
  139 |     std::function<double(double)> postProcessFn = nullptr) {
      |     ^~~~~~~~~~~~~
/home/circleci/project/src/RouteMap.cpp:138:55: note: ...following parameter 6 which has a default argument
  138 |     const wxString& requestKey, int gribIndex, double returnOnEmpty = NAN,
      |                                                       ^
/home/circleci/project/src/RouteMap.cpp: In function ‘double GetWeatherParameter(RouteMapConfiguration&, double, double, const wxString&, int, double, int)’:
/home/circleci/project/src/RouteMap.cpp:147:12: error: ‘postProcessFn’ was not declared in this scope
  147 |     return postProcessFn ? postProcessFn(value) : value;
      |            ^~~~~~~~~~~~~
/home/circleci/project/src/RouteMap.cpp:160:10: error: ‘postProcessFn’ was not declared in this scope
  160 |   return postProcessFn ? postProcessFn(value) : value;
      |          ^~~~~~~~~~~~~
/home/circleci/project/src/RouteMap.cpp: In function ‘double Swell(RouteMapConfiguration&, double, double)’:
/home/circleci/project/src/RouteMap.cpp:171:60: error: invalid user-defined conversion from ‘Swell(RouteMapConfiguration&, double, double)::<lambda(double)>’ to ‘int’ [-fpermissive]
  171 |       [](double height) { return height < 0 ? 0 : height; });
      |                                                            ^
/home/circleci/project/src/RouteMap.cpp:171:7: note: candidate is: ‘Swell(RouteMapConfiguration&, double, double)::<lambda(double)>::operator double (*)(double)() const’ (near match)
  171 |       [](double height) { return height < 0 ? 0 : height; });
      |       ^
/home/circleci/project/src/RouteMap.cpp:171:7: note:   no known conversion from ‘double (*)(double)’ to ‘int’
/home/circleci/project/src/RouteMap.cpp:139:5: note:   initializing argument 7 of ‘double GetWeatherParameter(RouteMapConfiguration&, double, double, const wxString&, int, double, int)’
  139 |     std::function<double(double)> postProcessFn = nullptr) {
      |     ^~~~~~~~~~~~~
/home/circleci/project/src/RouteMap.cpp: In function ‘double Gust(RouteMapConfiguration&, double, double)’:
/home/circleci/project/src/RouteMap.cpp:182:53: error: invalid user-defined conversion from ‘Gust(RouteMapConfiguration&, double, double)::<lambda(double)>’ to ‘int’ [-fpermissive]
  182 |       [](double gust) { return gust * 3.6 / 1.852; });  // Convert to knots
      |                                                     ^
/home/circleci/project/src/RouteMap.cpp:182:7: note: candidate is: ‘Gust(RouteMapConfiguration&, double, double)::<lambda(double)>::operator double (*)(double)() const’ (near match)
  182 |       [](double gust) { return gust * 3.6 / 1.852; });  // Convert to knots
      |       ^
/home/circleci/project/src/RouteMap.cpp:182:7: note:   no known conversion from ‘double (*)(double)’ to ‘int’
/home/circleci/project/src/RouteMap.cpp:139:5: note:   initializing argument 7 of ‘double GetWeatherParameter(RouteMapConfiguration&, double, double, const wxString&, int, double, int)’
  139 |     std::function<double(double)> postProcessFn = nullptr) {
      |     ^~~~~~~~~~~~~
make[2]: *** [CMakeFiles/weather_routing_pi.dir/build.make:280: CMakeFiles/weather_routing_pi.dir/src/RouteMap.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[2]: Leaving directory '/home/circleci/project/build'
make[1]: *** [CMakeFiles/Makefile2:402: CMakeFiles/weather_routing_pi.dir/all] Error 2
make[1]: Leaving directory '/home/circleci/project/build'
make: *** [Makefile:174: all] Error 2
```
-------------
The last one is similar.

@sebastien-rosset  These need to be fixed before I can publish it.

--- a/src/RouteMap.cpp
+++ b/src/RouteMap.cpp
@@ -66,6 +66,8 @@
 
 #include <stdlib.h>
 #include <math.h>
+#include <functional>
+#include <list>
 #include <map>
 
 #include "Utilities.h"

--- a/src/RouteMapOverlay.cpp
+++ b/src/RouteMapOverlay.cpp
@@ -1,9 +1,4 @@
 /***************************************************************************
- *
- * Project:  OpenCPN Weather Routing plugin
- * Author:   Sean D'Epagnier
- *
- ***************************************************************************
  *   Copyright (C) 2018 by Sean D'Epagnier                                 *
  *                                                                         *
  *   This program is free software; you can redistribute it and/or modify  *
@@ -20,11 +15,13 @@
  *   along with this program; if not, write to the                         *
  *   Free Software Foundation, Inc.,                                       *
  *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301,  USA.         *
- ***************************************************************************
- */
+ ***************************************************************************/
 
 #include <wx/wx.h>
 #include <wx/glcanvas.h>
+
+#include <functional>
+#include <list>
 
 #include "ocpn_plugin.h"
 #include "pidc.h"

